src/Alpha_complex/doc/Intro_alpha_complex.h

```diff
@@ -164,11 +164,11 @@ Table of Contents
  * <b>Requires:</b> \ref eigen &ge; 3.1.0 and \ref cgal &ge; 5.1.0.
  * 
  * A weighted version for Alpha complex is available (cf. Alpha_complex). It is like a usual Alpha complex, but based
- * on a <a href="https://doc.cgal.org/latest/Triangulation/index.html#title20">CGAL regular triangulation</a> instead
+ * on a <a href="https://doc.cgal.org/latest/Triangulation/index.html#TriangulationSecRT">CGAL regular triangulation</a> instead
  * of Delaunay.
  *
  * This example builds the CGAL weighted alpha shapes from a small molecule, and initializes the alpha complex with
- * it. This example is taken from <a href="https://doc.cgal.org/latest/Alpha_shapes_3/index.html#title13">CGAL 3d
+ * it. This example is taken from <a href="https://doc.cgal.org/latest/Alpha_shapes_3/index.html#AlphaShape_3DBuildingBasicAlphaShapesForManyPoints">CGAL 3d
```

Comment:
I think the link had already become wrong and is meant to link to `#AlphaShape_3DExampleforWeightedAlphaShapes`. We need to check 1 by 1 if they point to the right place, but indeed this proves that those links are better.

Reply:
I  think that based on my investigations it won't be difficult for you to check these.
(The links found looked reasonable to me).
Feel free to create a new PR or to changes this PR or to place a comment here with all the corrected values so I can add them to this PR.

Reply:
```suggestion
 * it. This example is taken from <a href="https://doc.cgal.org/latest/Alpha_shapes_3/index.html#AlphaShape_3DExampleforWeightedAlphaShapes">CGAL 3d
```